source/extensions/tracers/skywalking/skywalking_tracer_impl.cc

```diff
@@ -72,8 +72,11 @@ Tracing::SpanPtr Driver::startSpan(const Tracing::Config&, Tracing::TraceContext
           createSpanContext(toStdStringView(header_value_string)); // NOLINT(std::string_view)
       tracing_context = tracing_context_factory_->create(span_context);
     } catch (std::exception& e) {
-      ENVOY_LOG(warn, "New SkyWalking Span/Segment cannot be created for error: {}", e.what());
-      return std::make_unique<Tracing::NullSpan>();
+      ENVOY_LOG(
+          warn,
+          "New SkyWalking Span/Segment with previous span context cannot be created for error: {}",
+          e.what());
+      tracing_context = tracing_context_factory_->create();
```

Comment:
If there is no valid propagation header, I think we should downgrade to the case where propagation header is no exist. In that case, the `decision` will be used to decide to create null span or valid span.

Reply:
done,thanks,